decred/decred/dcr/dcrdata.py

```diff
@@ -1125,6 +1143,9 @@ def purchaseTickets(self, keysource, utxosource, req):
                 UTXOs. The filterFunc is an optional function to filter UTXOs,
                 and is of the form func(UTXO) -> bool.
             req account.TicketRequest: the ticket data.
+            relayFee (int): Transaction fees in atoms per kb.
```

Comment:
It's worth mentioning here that for the ticket (but not the split tx), the `relayFee` is overridden by the `req.txFee`.

Reply:
hmm, why have a relayFee parameter at all? Can we remove one of them?

Reply:
Like you discovered, the ticket fee and split tx fee may need different fees. If the user wants to pay the low fee on the split tx, you apparently need to set the ticket separately.

Reply:
I see. But what I mean is that we have the split tx fee defined in the req already, so we don't need it again in the constructor, which is overridden by the req.txFee. Maybe I'm not seeing something obvious here? We just need to two fee rates, which are in the req already.